pkg/utils/mathutil/mathutil.go

```diff
@@ -0,0 +1,9 @@
+package mathutil
+
+// Max returns the larger of a and b.
+func Max(a, b int) int {
```

Comment:
Why not using math.Max https://pkg.go.dev/math#Max

Reply:
The parameters and return value of `math.Max` are all `float64`. `mathutil.Max(a, b)` should written as `int(math.Max(int(a), int(b)))`, there are too many type conversions.

Reply:
OK, if you prefer this way, I think it's unnecessary to put it into another package, just make it an inner function, it'll be easy to read.